libs/core/kiln_ai/utils/open_ai_types.py

```diff
@@ -84,11 +84,28 @@ class ChatCompletionAssistantMessageParamWrapper(TypedDict, total=False):
     """The tool calls generated by the model, such as function calls."""
 
 
+class ChatCompletionToolMessageParamWrapper(TypedDict, total=False):
+    content: Required[Union[str, Iterable[ChatCompletionContentPartTextParam]]]
+    """The contents of the tool message."""
+
+    role: Required[Literal["tool"]]
+    """The role of the messages author, in this case `tool`."""
+
+    tool_call_id: Required[str]
+    """Tool call that this message is responding to."""
+
+    kiln_task_tool_data: Optional[str]
+    """The data for the Kiln task tool that this message is responding to.
+    
+    Formatted as `<project_id>,<tool_id>,<task_id>,<run_id>`
```

Comment:
We use `::` delimiters. 

Good to follow the convention, but more importantly commas might be allowed in IDs so might be serious (not positive but let's follow convention anyways and save 15 seconds checking 😆 ). I know we allow a bunch of chars even though they are usually numbers.